static/src/stylesheets/module/onward/_essential-read.scss

```diff
@@ -3,8 +3,14 @@
     margin-bottom: $gs-baseline;
 
     .fc-container__header__title {
+        font-size: 2.25rem;
```

Comment:
@zeftilldeath these should use the guss typography mixins (sorry, missed this PR!)